javascript/ql/src/semmle/javascript/frameworks/ClientRequests.qll

```diff
@@ -152,6 +219,24 @@ private class AxiosUrlRequest extends ClientRequest::Range {
       result = getOptionArgument([0 .. 2], name)
     )
   }
+
+  string getResponseFormat() {
+    exists(DataFlow::Node option | option = getOptionArgument([0 .. 2], "responseType") |
+      result = option.getStringValue()
+      or
+      not exists(option.getStringValue()) and
+      result = ""
```

Comment:
Can we add a test for the `""` response? It is a general gotcha to accidentally remove the "unknown" result.